git_machete/utils.py

```diff
@@ -20,25 +20,25 @@
 ascii_only: bool = False
 
 
-def excluding(iterable: Iterable[T], s: Iterable[T]) -> List[T]:
+def get_excluding(iterable: Iterable[T], s: Iterable[T]) -> List[T]:
     return list(filter(lambda x: x not in s, iterable))
 
 
-def flat_map(func: Callable[[T], List[T]], iterable: Iterable[T]) -> List[T]:
+def get_flat_map(func: Callable[[T], List[T]], iterable: Iterable[T]) -> List[T]:
```

Comment:
I'd make exception for `flat_map`, `excluding`, `map_truthy_only`, `tupled` &mdash; it's pretty clear that these function are meant to be side-effect-free (also, `map` is already a verb, so another `get` verb feels pretty redundant)